test/src/main/java/org/apache/accumulo/test/zookeeper/fate/ZooLockIT.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.accumulo.test.fate.zookeeper;
+package org.apache.accumulo.test.zookeeper.fate;
```

Comment:
`fate.zookeeper` makes slightly more sense than `zookeeper.fate` because 1) it avoids one relocation, which is always nicer on git archeology, and 2) it more closely matches the package name of the corresponding code it is testing.

There's technically no reason why it should have `fate` in the package name at all. That's leftover from this code being in a separate jar. It's not fate-specific code. But, until the code itself is moved, the test code should probably stay in a package that resembles the code it is testing.

Reply:
I have included this change in the most recent commit.